src/Language/TypeScript/Resolution.hs

```diff
@@ -83,39 +83,40 @@ resolveNonRelativePath :: ( Has (Modules address value) sig m
                           , Has (Resumable (BaseError ResolutionError)) sig m
                           , Has Trace sig m
                           )
-                       => FilePath
+                       => Path.RelDir
```

Comment:
The name of this function is a little weird given that it says “NonRelative” but takes a RelDir. Can you add a TODO about renaming it?

Reply:
Fixed!